### ISSUE TITLE
Rubber trees stop produce rubber after server restart

### ISSUE THREAD

**Issue by Akiraca** [GregTechCE/GregTech#689]
Although this issues is posted several days ago it hasn't been fixed yet in the latest build.

Can be easily reproduce by restarting server with only GTCE & code chicken lib installed.

**Comment by Archengius** [comment]
cannot reproduce in dev environment or link this bug with restarts

**Comment by Akiraca** [comment]
Can the issue be reproduced in normal gaming environment (minecraft with forge) on your computer?

**Comment by Beetrootmonkey** [comment]
Sorry to necro, but I can confirm this behaviour. I suspect that this is an issue with writing/reading the block state. That would mean this is also an issue when unloading/loading chunks that contain rubber trees, I guess.

**Comment by pyure** [comment]
Beetrootmonkey, are you trying to raise a new concern with this issue, or did you just not notice that its resolved and closed?

**Comment by Beetrootmonkey** [comment]
My apologies. I was very tired when I wrote my previous comment.
While I did notice that this issue had been closed, I didn't realized it had been solved. It looked to me like you simply couldn't reproduce the bug and thus closed this issue.

Plus, if you only resolved this issue 3 days ago, the GregtechCE version on my server certainly isn't up-to-date (it's 7 days old). I should have made sure to check that first.

Sorry for the confusion. Have a nice day.
